src/angular/4-adding-routing/lazyload-route.html

```diff
@@ -26,16 +26,16 @@
       <p>Look at me!</p>
       <ul class="nav">
         <li routerLinkActive="active">
-          <a routerLink="/about" >About</a>
+          <a routerLink="/about">About</a>
         </li>
         <li routerLinkActive="active">
-          <a routerLink="boop">/boop - I will reroute to /about</a>
+          <a routerLink="colophon">/colophon - I will reroute to /about</a>
```

Comment:
what is a colophon? 👀

Reply:
could we add something related to about instead? like 'about-us'

Reply:
It’s another term [used on websites for about pages](https://indieweb.org/colophon). I’ll change it to `about-us`.

Reply:
Do you feel strongly about this one? I wanted to use a different word than `about` so when explaining it, the word “about” isn’t used multiple times, e.g. “the colophon page will redirect to about” vs. “the about-us page will redirect to about.” If you do think it needs changing, I’ll update it to something else.

Reply:
I've never seen colophon before (and it could be because English is not my main language), but I feel if we use uncommon words or expressions, the content becomes less accessible for non-native English speakers. It could be something other than about, even `foo` would be ok to me because it's spot on, colophon looks like something I should know the meaning of but I don't 😬

Reply:
As much as possible, I think it’s best to keep examples grounded in reality because it’s easier for people to relate to real examples. I’m going to change this to redirect `/team`. Thanks for the feedback!